src/modules/utils/simpleshell/SimpleShell.cpp

```diff
@@ -870,6 +892,8 @@ void SimpleShell::get_command( string parameters, StreamOutput *stream)
             THEROBOT->inch_mode ? 20 : 21,
             THEROBOT->absolute_mode ? 90 : 91,
             get_spindle_state()?'3':'5',
+            get_mist_state()?'7':'9',
+//            get_flood_state()?'8':'9',     // TODO How to combine in the same printf var ???
```

Comment:
is it always either M7 or M8 (or M9)

Reply:
From Linuxcnc/GRBL yes.

I haven't updated Gcode parser, only the data variable.

This data is triggered with the switch.mist.enable without other code change.

Br